### PR TITLE
DOCS-2488 improve universal toolbar doc

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
From spreadsheet line 6: https://docs.google.com/spreadsheets/d/1KST5bprUIbQsTybYIF5mB-IMxR2AWpQHa2nVK4aHSwU/edit?ts=5b46a097#gid=1051349989

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
I left it in both places since the mule4 user guide topic is all about changes to studio...

File: anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc [hunk 1]
```diff
@@ -12,6 +12,17 @@ If your organization has external identity configured to access Anypoint Platfor
 
 You can configure multiple users to authenticate in Anypoint Platform.
 
+In every operation related to the Platform, you'll see a quick-change toolbar that allows you to change accounts and if applicable, the business group. You can also add accounts from the toolbar.
+
+For example, when creating a new Mule project (File > New > Mule Project), if you select an API implementation from the Design Center, the toolbar displays at the top of the selection dialog:
+
+image:/mule4-user-guide/v/4.1/_images/studio-credentials.png[Studio credentials when creating a new Mule Project]
```
Comment (fermujica) [inline anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc]: Images for Studio must go in the anypoint-studio/v/7.1/_images directory.
And the sytanx to call them is simply:
`image:studio-credentials.png[Studio credentials when creating a new Mule Project]`

File: anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc [hunk 2]
```diff
@@ -12,6 +12,17 @@ If your organization has external identity configured to access Anypoint Platfor
 
 You can configure multiple users to authenticate in Anypoint Platform.
 
+In every operation related to the Platform, you'll see a quick-change toolbar that allows you to change accounts and if applicable, the business group. You can also add accounts from the toolbar.
+
+For example, when creating a new Mule project (File > New > Mule Project), if you select an API implementation from the Design Center, the toolbar displays at the top of the selection dialog:
```
Comment (fermujica) [inline anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc]: Drop the 'the' from Design Center.

File: mule4-user-guide/v/4.1/intro-studio.adoc [hunk 3]
```diff
@@ -23,9 +23,15 @@ You can now navigate quickly to the XML from the visual view:
 
 == Anypoint Platform Account Management
 
-MuleSoft also improved how you manage your Anypoint Platform accounts, making it easier to publish and consume different assets from within Studio and to see which business group you are working with. We enabled this by creating a universal user toolbar that appears in every operation related to the Platform. This toolbar enables you to switch between users and organizations, or even add accounts.
+In every operation related to the Platform, you'll see a quick-change toolbar that allows you to change accounts and if applicable, the business group. You can also add accounts from the toolbar.
 
-image:studio-credentials.png[Studio Credentials]
+For example, when creating a new Mule project (File > New > Mule Project), if you select an API implementation from the Design Center, the toolbar displays at the top of the selection dialog:
```
Comment (fermujica) [inline mule4-user-guide/v/4.1/intro-studio.adoc]: Drop the 'the' from Design Center.

File: anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [reply anypoint-studio/v/7.1/set-credentials-in-studio-to.adoc]: Oh, so we have to copy images into each version directory? Will do, thanks!

**Pull request merged**